source/tools/VirtualAudioSink.h

```diff
@@ -78,6 +79,12 @@ class VirtualAudioSink : public AudioSinkBase
         mNextHardwareReadTimeNanos = 0;
         mFramesConsumed = 0;
         mStartTimeNanos = 0;
+
+        if (Controller.isUtilClampSupported()) {
+            mSampleRate = sampleRate;
+            Controller.setUtilClampMin(1024);
```

Comment:
Why is this being done? it seems that will boost a non-audio thread, which just messes with the governor.
Maybe set mCurrentUtilClamp at the beginning of the rt audio thread loop.

Reply:
I removed and I set UtilClamp min before audioCallback loop, but the results are not good (some times worse than the synthmark original version)

Reply:
This is still being called in the open. We can fix this in a future PR.